### ISSUE TITLE
Metamask to Terra station 

### ISSUE THREAD

**Issue by CryptoJayFPS** [terra-money/bridge-web-app#220]
![Classic transfer to Terra Station](https://user-images.githubusercontent.com/112282667/187048117-809d1584-540b-4e6d-9447-fc7bd6aaf3d7.png)

**Comment by votrix81** [comment]
Did you receive your coins? I am also waiting for some the last 4 hours.

**Comment by CryptoJayFPS** [comment]
I got mines, this web page never replied to my issue but I recovered 2,200,000 that got lost in the Terra bridge portal.Sent from my iPhoneOn 21 Sep 2022, at 11:56, votrix81 ***@***.***> wrote:﻿
Did you receive your coins? I am also waiting for some the last 4 hours.

**Comment by votrix81** [comment]
How did you recover them? I need to do the same because mine are totally stuck/lost.

> I got mines, this web page never replied to my issue but I recovered 2,200,000 that got lost in the Terra bridge portal.Sent from my iPhoneOn 21 Sep 2022, at 11:56, votrix81 ***@***.***> wrote:﻿ Did you receive your coins? I am also waiting for some the last 4 hours. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by CryptoJayFPS** [comment]
Yeah it’s not hard, and be carful with these guys they will ask you for the key phrase seems like everyone is a scammer. In my case I only know how to recover them is you moved it from metamask to to terra bridge attempting to swap them to terra station. Other then that I can’t help this was my situation.Sent from my iPhoneOn 21 Sep 2022, at 12:44, votrix81 ***@***.***> wrote:﻿
How did you recover them? I need to do the same because mine are totally stuck/lost.

I got mines, this web page never replied to my issue but I recovered 2,200,000 that got lost in the Terra bridge portal.Sent from my iPhoneOn 21 Sep 2022, at 11:56, votrix81 @.> wrote:﻿ Did you receive your coins? I am also waiting for some the last 4 hours. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: @.>

**Comment by votrix81** [comment]
Could you tell me how you did it please? Did you use Shuttle or Wormhole?

> Yeah it’s not hard, and be carful with these guys they will ask you for the key phrase seems like everyone is a scammer. In my case I only know how to recover them is you moved it from metamask to to terra bridge attempting to swap them to terra station. Other then that I can’t help this was my situation.Sent from my iPhoneOn 21 Sep 2022, at 12:44, votrix81 ***@***.***> wrote:﻿ How did you recover them? I need to do the same because mine are totally stuck/lost. I got mines, this web page never replied to my issue but I recovered 2,200,000 that got lost in the Terra bridge portal.Sent from my iPhoneOn 21 Sep 2022, at 11:56, votrix81 @.> wrote:﻿ Did you receive your coins? I am also waiting for some the last 4 hours. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: @.> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by CryptoJayFPS** [comment]
I used the wormhole I’m at work now in a business meeting but I promise I’ll help you shortly.Sent from my iPhoneOn 21 Sep 2022, at 13:40, votrix81 ***@***.***> wrote:﻿
Could you tell me how you did it please? Did you use Shuttle or Wormhole?

Yeah it’s not hard, and be carful with these guys they will ask you for the key phrase seems like everyone is a scammer. In my case I only know how to recover them is you moved it from metamask to to terra bridge attempting to swap them to terra station. Other then that I can’t help this was my situation.Sent from my iPhoneOn 21 Sep 2022, at 12:44, votrix81 @.> wrote:﻿ How did you recover them? I need to do the same because mine are totally stuck/lost. I got mines, this web page never replied to my issue but I recovered 2,200,000 that got lost in the Terra bridge portal.Sent from my iPhoneOn 21 Sep 2022, at 11:56, votrix81 @.> wrote:﻿ Did you receive your coins? I am also waiting for some the last 4 hours. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: @.> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: @.>

**Comment by votrix81** [comment]
Thanks. Since I used Shuttle I used classic-bridge.terra.money and I think you used the other one for wormhole which has a redeem button. I don't see a redeem/retrieve button for the Shuttle one.Mine were eith-wrapped. But any ideas you can offer would be great. Best.

> I used the wormhole I’m at work now in a business meeting but I promise I’ll help you shortly.Sent from my iPhoneOn 21 Sep 2022, at 13:40, votrix81 ***@***.***> wrote:﻿ Could you tell me how you did it please? Did you use Shuttle or Wormhole? Yeah it’s not hard, and be carful with these guys they will ask you for the key phrase seems like everyone is a scammer. In my case I only know how to recover them is you moved it from metamask to to terra bridge attempting to swap them to terra station. Other then that I can’t help this was my situation.Sent from my iPhoneOn 21 Sep 2022, at 12:44, votrix81 @.> wrote:﻿ How did you recover them? I need to do the same because mine are totally stuck/lost. I got mines, this web page never replied to my issue but I recovered 2,200,000 that got lost in the Terra bridge portal.Sent from my iPhoneOn 21 Sep 2022, at 11:56, votrix81 @.> wrote:﻿ Did you receive your coins? I am also waiting for some the last 4 hours. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: @.> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: @.> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
